src/utils/webgl.js

```diff
@@ -34,3 +34,8 @@ export function patchWebGLRenderingContext() {
     };
   }
 }
+
+export function getWebGLVersion(renderer) {
```

Comment:
Nit: Just my preference. I prefer taking `gl`, rather than `renderer`, as an argument to reduce the dependency with Three.js because the file name is `webgl.js`. No blocking issue, just my preference.